frontend/ui/src/pages/Channels/Facebook/RequirementsDialog.tsx

```diff
@@ -47,3 +47,51 @@ export const RequirementsDialog = (props: RequirementsDialogProps) => {
     </SettingsModal>
   );
 };
+
+export const SmsWhatsappDialogue = (props: RequirementsDialogProps) => {
+  return (
+    <SettingsModal style={{maxWidth: '582px'}} title="Connect Messenger" close={props.onClose}>
+      <div className={styles.backgroundContainer}>
+        <div className={styles.container}>
+          <button className={styles.closeButton} onClick={props.onClose}>
+            <img src={close} />
+          </button>
+          <div className={styles.inviteWrapper}>
+            <h1 className={styles.headline}>Connect with Twilio First</h1>
+            <p className={styles.firstMessage}>
+              Before you connect a number for SMS or Whatsapp, you must add a{' '}
+              <a
+                href="https://support.twilio.com/hc/en-us/articles/223136027-Auth-Tokens-and-How-to-Change-Them"
+                target="_blank"
+                rel="noreferrer">
+                Twilio Auth Token{' '}
```

Comment:
Nitpicking: this space extends the underscore by 1  so it isn't align with the phrase anymore